pyproject.toml

```
@@ -44,7 +44,9 @@ dependencies = [
     "backoff",
     "monotonic",
     # for logging, tracebacks, printing, progressbars
-    "rich <= 13.0.1"
+    "rich <= 13.0.1",
+    # for CLI
+    "typer >= 0.6.0, < 0.8.0"
```

```suggestion
    "typer >= 0.6.0, < 0.8.0"
```
Perhaps we can be more flexible on this upper bound now that 0.9.0 is out?

---

According to the `0.9.0` just the default/recommended argument definition changed, but everything's backwards compatible, so we can probably just `>=0.6.0,<1.0.0`?